src/swagger.js

```diff
@@ -90,6 +91,9 @@ export const swaggerDocument = {
     '/api/v1/products': {
       post: createProduct,
     },
+    '/api/v1/products/unprotected?page={page}&size={size}': {
```

Comment:
same here, change this route from we can have /api/v1/products/public instead of  api/v1/products/unprotected